src/pl/plr.spec

```diff
@@ -3,10 +3,14 @@ Name:           plr-hawq
 Version:        08.03.00.14
 Release:		0
 Prefix:         /usr/local
-License:		GPL
+License:		Apache
+BuildRequires:  R-devel >= 3.1.0
```

Comment:
No, we should not use '=' since we are installing R and R-devel via YUM. Users might have higher version YUM repo. It will be very hard to find the correct YUM repo.